client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2156,6 +2156,20 @@ Undocumented.prototype.transferStatus = function( siteId, transferId ) {
 };
 
 /**
+ * Get user suggestions for a site.
+ *
+ * @param {int}       siteId            The site ID
+ * @returns {Promise}
+ */
+Undocumented.prototype.getUserSuggestions = function( siteId ) {
+	debug( '/users/suggest/ query' );
+
+	return this.wpcom.users().suggest( {
```

Comment:
@aduth For editor mentions, we won't actually need to do any filtering as we need all user suggestions initially. I'd agree that it could be useful in the future, but I'm wondering if it wouldn't be more appropriate to add it in if / when it's determine that it's needed?

@gwwar Ah, I see and that makes sense. I've addressed this in [this commit](https://github.com/Automattic/wp-calypso/pull/9969/commits/26c73ed6c9f298a0e86ff3ee4ad681932411d376).

Thanks all.